config/locales/en/course/discussion/post.yml

```diff
@@ -0,0 +1,5 @@
+en:
+  course:
+    discussion:
+      post:
+        confirm-delete-post: 'Are you sure that you want to delete this post?'
```

Comment:
This text is actually shared between comments and forum posts, so it could be a bit confusing to see the word 'post' when trying to delete a comment.

One option is to detect the post type and change the wording accordingly.

The easy way out is to just ask "Are you sure you want to delete this?"

What do you all think? @weiqingtoh @allenwq